next.config.js

```diff
@@ -3,6 +3,10 @@
  */
 const UI_VERSION = "4.2.0";
 
+const LOG_FETCH_REQUEST_TIME = true;
+
+const LOG_GET_SERVER_SIDE_REQUEST_TIME = true;
```

Comment:
So these are just temporary instead of using environment variables, I assume. I see the timings when I try it.

Reply:
oh I'm not sure I understand. I hadn't intended originally to use env variables to control this, but we could I guess.

Temporary in what way? Like we would remove these at some point in the future?

Reply:
I think environment variables would be good, if they can be set up in a demo. If this goes into dev, I just didn’t want these enabled by default, and have every demo and local instance logging times.

Reply:
Is it possible to make these false by default and then enable them selectively for a demo or whatever? Also I can just make these env vars. I'm not sure how I'd enable them on a demo though.

Reply:
I’m not sure how much value my suggestion has, now. I don’t _think_ you can just deploy a demo with different environment variables, though I could be wrong. So if you have to change the branch to change environment variables (like in a docker file), then that’s no better than just setting these variables in Javascript in a branch.